### PR TITLE
Add module website_sale_product_legal.

### PR DESCRIPTION
**Pull request by yajo**
# Legal terms per product

This module was written to extend the functionality of e-commerce to support setting specific legal terms per product and allow you to inform your online buyers that they are subjected to those when buying that product.

CC @rafaelbn.

### CODE REVIEW

**Comment by rafaelbn** [comment]
If want to sell products for over +18 years old or similar
Absolutely :+1: , super but I think we must improve UX when clicking in "By buying this product, you are accepting its custom legal terms." it must show a pop-up and not open a new tab.
cc @dgzurita @sergio-incaser @carlos-incaser @janetrp @JavierIniesta
Tested in runbot with Rum HAvanna http://3119077-72-1f6d68.runbot1.odoo-community.org/shop/product/rum-53

**Comment by yajo** [comment]
good idea

File: website_sale_product_legal/views/templates.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Antiun Ingeniería, S.L. - Jairo Llopis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<template id="legal_terms">
+    <t t-call="website.layout">
```
Comment (sergio-teruel) [inline website_sale_product_legal/views/templates.xml:9]: add div container to limit width

File: website_sale_product_legal/views/product_template_view.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<openerp>
+<data>
+
+<record id="product_template_form" model="ir.ui.view">
+    <field name="name">Add legal terms</field>
+    <field name="model">product.template</field>
+    <field name="inherit_id" ref="product.product_template_form_view"/>
+    <field name="arch" type="xml">
+        <xpath expr="//group[@string='Sale Conditions']">
```
Comment (carlosdauden) [inline website_sale_product_legal/views/product_template_view.xml]: Use name='sale_condition' instead string

Comment (yajo) [inline website_sale_product_legal/views/product_template_view.xml]: Why?

Comment (pedrobaeza) [inline website_sale_product_legal/views/product_template_view.xml]: Because in v9 this is not allowed, so we save one step for future migrations.

**Comment by yajo** [comment]
All should be fixed.

**Comment by rafaelbn** [comment]
When you click on "custom legal terms" it doesn't make anything, neither legal page nor pop-up.
I tested in chrome and firefox, doesn't work pop-up.

**Comment by yajo** [comment]
It should work now.

**Comment by janetrp** [comment]
in firefox custom legal terms is not working, nothing happens
2015-11-13 17_23_38-auriculares negros _ localhost

**Comment by yajo** [comment]
only works if logged in. Note to self: add  `<link rel='stylesheet' href="/web/static/lib/jquery.ui/css/smoothness/jquery-ui-1.9.1.custom.css"/>`  to assets

**Comment by rafaelbn** [comment]
I have tested again in both mode logout and logged. It's working fine, looks ready to merge :+1: , thanks!

**Comment by janetrp** [comment]
Tested again and works fine :+1:

**Comment by JavierIniesta** [comment]
Tested in runbot :+1:

**Comment by dgzurita** [comment]
Tested in runbot :+1:

**Comment by yajo** [comment]
You might want to review #78 now, thanks.

**Pull request merged**